Blish HUD/Blish HUD.csproj

```diff
@@ -210,11 +216,11 @@
     <PackageReference Include="Humanizer.Core.fr" Version="2.6.2" PrivateAssets="all" />
     <PackageReference Include="JsonSubTypes" Version="1.6.0" PrivateAssets="all" />
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="all" />
-    <PackageReference Include="MonoGame.Content.Builder" Version="3.7.0.9" PrivateAssets="all" />
-    <PackageReference Include="MonoGame.Extended" Version="[3.7.0]" />
-    <PackageReference Include="MonoGame.Extended.Content.Pipeline" Version="3.7.0" PrivateAssets="all" />
-    <PackageReference Include="MonoGame.Framework.WindowsDX" Version="[3.7.1.189]" />
-    <PackageReference Include="Newtonsoft.Json" Version="[12.0.2]" />
+    <PackageReference Include="MonoGame.Content.Builder.Task" Version="3.8.0.1641" />
+    <PackageReference Include="MonoGame.Extended" Version="3.8.0" />
+    <PackageReference Include="MonoGame.Extended.Content.Pipeline" Version="3.8.0" />
+    <PackageReference Include="MonoGame.Framework.WindowsDX" Version="3.8.0.1641" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
Here and elsewhere where the previous versions were listed with brackets (for example, see below), we'll need to brackets returned to avoid causing problems with modules and their dependency expectations.

`<PackageReference Include="Newtonsoft.Json" Version="[12.0.2]" />`

Reply:
Resolved, although I don't understand the implications of pinning vs. not pinning the versions.